pilot/lib/pilot/tester_manager.rb

```diff
@@ -43,6 +40,26 @@ def add_tester(options)
       end
     end
 
+    def add_tester_to_groups(tester: nil, app: nil)
+      default_external_group = app.default_external_group
+      if default_external_group.nil? && config[:groups].nil?
```

Comment:
Is `add_tester_to_groups` meant to be a private method of TesterManager? I only ask because it takes a `tester` and an `app` but also references the `config` hash. The pattern in this file seems to be that public methods reference `config` and take an `options` argument. Private methods take specific parameters but don't reference `config`.

My thought: mark this method as private, and pass in the `groups` as an argument.